docs/guides/formats.md

```diff
@@ -0,0 +1,62 @@
+# Data formats
```

Comment:
I think there is a need to clarify that different data formats can be used with the standard, but agree maybe it doesn't need to go into this level of detail here.

Reply:
Removed from guides page, and will move it to a readme on the github page but not published on documentation.

Reply:
I removed guides/ddh-rdl.md to an internal readme on github, but not in public documentation:
https://github.com/GFDRR/rdl-standard/blob/docs.mat/internal_guide_rdl_on_WBdataCatalog.md

It is clear that this file is very focussed on RDL implementation, which only WB admins can use.
[formats](https://github.com/GFDRR/rdl-standard/tree/docs.mat/docs/guides/formats.md) and 
[packaging](https://github.com/GFDRR/rdl-standard/tree/docs.mat/docs/guides/packaging.md) are a bit more equivocal I think, and have broader information for people using other implementations.
To discuss with @matamadio too - we could make the guidance more general and retain it on public documentation, or make internal.

Reply:
> I think there is a need to clarify that different data formats can be used with the standard,

Agreed! Looking at the introductory paragraph on the [homepage](https://rdl-standard.readthedocs.io/en/docs.mat/), I think that we should clarify the purpose and applicability of the standard upfront. If we make the distinction between RDLS metadata and risk datasets very clear from the beginning of a reader's journey, I think that will make it easier to structure the rest of the documentation, something like this (changes in italics):

> The Risk Data Library Standard (RDLS) is an open data standard *for metadata about risk datasets. It can be used to publish metadata for risk datasets provided in any format. RDLS metadata* makes it easier to *identify and* work with disaster and climate risk data*sets*. It provides a common description of the data used and produced in risk assessments, including hazard, exposure, vulnerability, and modelled loss, or impact, data.

If you want to keep the guidance on formats and packaging for risk datasets (rather than for RDLS metadata) within the RDLS docs, I would put them under a separate section of the docs and add a sentence to the home page along the lines of:

> In addition to providing the metadata standard, RDLS also provides [guidance on how to structure and format risk datasets]().

Reflecting on the above, we should probably also explain the meaning of metadata (a set of data that describes and gives information about other data) fairly early in the introductory content.